janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/diskstorage/hbase/HBaseStoreManagerMutationTest.java

```diff
@@ -127,4 +129,33 @@ public void testKCVMutationToPuts() throws Exception {
         Collections.sort(deleteColumns);
         Assert.assertArrayEquals(expectedColumnDelete.toArray(), deleteColumns.toArray());
     }
+
+    @Test
+    public void testMutationToPutsTTL() throws Exception{
```

Comment:
The test is failing `org.junit.internal.ArrayComparisonFailure: arrays first differed at element [1]; expected:<2147484000> but was:<-2147483296>`.